protected_tags.go

```diff
@@ -46,6 +46,8 @@ type ProtectedTag struct {
 type TagAccessDescription struct {
 	AccessLevel            AccessLevelValue `json:"access_level"`
 	AccessLevelDescription string           `json:"access_level_description"`
+	UserID                 *int             `url:"user_id,omitempty" json:"user_id,omitempty"`
+	GroupID                *int             `url:"group_id,omitempty" json:"group_id,omitempty"`
```

Comment:
I cannot see in the docs that these fields need to be here. Can you point me to somewhere (in the code) that shows these fields exist in this struct?

Additionally in a response struct like this we don't use pointers for build-in types and URL tags.